apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -145,6 +145,13 @@ export default function LearningGoal({
 
   // Callback to retrieve understanding data from EvidenceLevels
   const radioButtonCallback = radioButtonData => {
+    analyticsReporter.sendEvent(EVENTS.TA_RUBRIC_EVIDENCE_LEVEL_SELECTED, {
+      ...(reportingData || {}),
```

Comment:
This allows us to pass through the other reporting data from this learning goal such as the unit, course, and level